provider/src/main/scala/com/actian/spark_vector/provider/Job.scala

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2016 Actian Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.actian.spark_vector.provider
+
+import com.actian.spark_vector.datastream.{ VectorEndpoint, VectorEndpointConf }
+import com.actian.spark_vector.vector.ColumnMetadata
+
+import play.api.libs.json.{ Format, JsObject, Json, Reads, Writes }
+
+private[provider] case class LogicalType(`type`: String,
```

Comment:
can we avoid using the backtick by just replacing `type` with `name` for instance ?
`LogicalType(name, precision, scale)` -- imo here `name` refers to the LogicalType's name